scripts/rename-tinyagentos.sh

```
@@ -0,0 +1,776 @@
+fi
+
+if [[ ! -d "$REPO_DIR/.git" ]] && [[ ! -f "$REPO_DIR/.git" ]]; then
+    echo "ERROR: $REPO_DIR is not a git repository. Use --repo-dir."
+    exit 1
+fi
+
+cd "$REPO_DIR"
+
+# ── Undo mode ──────────────────────────────────────────────────────────────
+if $UNDO; then
+    BACKUP_DIR="$(ls -dt .rename-backup-* 2>/dev/null | head -1 || true)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)